service/controller/release/resource/status/cluster.go

```diff
@@ -67,14 +66,17 @@ func (r *Resource) getCurrentTenantClusters(ctx context.Context) ([]tenantCluste
 
 // Returns a list of AWS clusters according to the awscluster resource.
 func (r *Resource) getCurrentAWSClusters(ctx context.Context) ([]tenantCluster, error) {
-	awsClusters := capiv1alpha2.ClusterList{}
-	err := r.k8sClient.CtrlClient().List(ctx, &awsClusters)
+	awsClusters, err := r.listPartialObjectMetadata(ctx, metav1.GroupVersionKind{
+		Group:   "cluster.x-k8s.io",
```

Comment:
This broke release-operator. There is no `AWSClusters` on that group. AWSClusters belong to `infrastructure.cluster.x-k8s.io`.

Reply:
I'm testing a fix here https://github.com/giantswarm/release-operator/pull/379

Reply:
Sorry about that. I should have tested on AWS and Azure too. Thanks for fixing it. Let me know if I can help.